scripts/Tools/xmlchange

```diff
@@ -143,7 +143,7 @@ if ( ! defined($settinglist) ) {
    $idlist{$opts{'id'}} = $opts{'val'};
 } else {
    foreach my $varval ( split( /,/, $settinglist ) ) {
-      if ( $varval =~ /^([a-zA-Z0-9_]+)=([^,=]+)$/ ) {
+       if ( $varval =~ /^(\w+)=(.*+)$/ ){
```

Comment:
This change looks unrelated to the rest of the pull request, right? Ideally this would be in a separate pull request. But at the very least can you document the purpose of this change and the testing you have done to ensure that it is correct?